index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Youtube Project</title>
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css"
+    />
+  </head>
+  <body>
+    <nav id="nav-bar">
+      <div id="logo">
+        <span>
+          <i class="bar-icon fas fa-bars"></i>
+        </span>
+        <img class="image-logo" src="img/logo3.jpg" alt="logo-image" />
+      </div>
+      <div class="search-bar">
+        <input type="text" placeholder="Search" /><button id="search-btn">
+            <i class="fas fa-search"></i>
+          </button>
+     </div>
+      <div class="icons">
+        <span><i class="fas fa-video "></i></span>
+        <span><i class="fas fa-th "></i></span>
+        <span><i class="fas fa-bell "></i></span>
+        <span><i class="fas fa-user-circle "></i></span>
+      </div>
+    </nav>
+
+    <main id="main-box">
+      <section id="main-section">
+        <!-- <iframe src="" frameborder="0"></iframe> -->
+        <iframe
+          src="https://www.youtube.com/embed/PkZNo7MFNFg"
+          allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        ></iframe>
+        <h3>Learn JavaScript- Full Course for Beginners</h3>
+        <div style="display: flex; justify-content: space-between;">
+          <h2 style="font: 14px sans-serif;">
+            2,963,622 views Dec 10 2018
+          </h2>
```

Comment:
- [ ] Try to give a little more space to the top and bottom of this section. The elements look too cramped.
![Screenshot from 2020-03-27 12-08-00](https://user-images.githubusercontent.com/51926937/77786616-ba4d0f00-7023-11ea-81a8-09a7fa757080.png)